dfget/core/downloader/power_client.go

```diff
@@ -254,9 +258,24 @@ func (cw *ClientWriter) write(piece *Piece, startTime time.Time) error {
 	_, err := io.Copy(buf, piece.RawContent())
 	buf.Flush()
 
+	go cw.sendSuccessPiece(piece, time.Since(startTime))
 	return err
 }
 
+func (cw *ClientWriter) sendSuccessPiece(piece *Piece, cost time.Duration) {
+	cw.api.ReportPiece(piece.SuperNode, &types.ReportPieceRequest{
+		TaskID:     piece.TaskID,
+		Cid:        cw.Cfg.RV.Cid,
+		DstCid:     piece.DstCid,
+		PieceRange: piece.Range,
+	})
+	if cost.Seconds() > 2.0 {
```

Comment:
Just curious.  What is the necessity of this judgment?

Reply:
output which piece download takes more time